### PR TITLE
Release 2.8.3

### PR DESCRIPTION
**Pull request by sergiou87**
## Description

Looking for the PR for the upcoming v2.8.3 production release? Well you've just found it, congratulations! :tada:

This is based on the 2.8.2 release (tag [release-2.8.2](https://github.com/desktop/desktop/releases/tag/release-2.8.2)) and includes:
- Enable cross self-update to arm64: #12332 
- Fix of caching issue cloning renamed repos: https://github.com/desktop/desktop/pull/12308
- Fix for expanding whole files which could cause duplicated lines: https://github.com/desktop/desktop/pull/12251
- Fix for settings UI when git email is too long: https://github.com/desktop/desktop/pull/12159
- Improvement to create branches from filter textfield: https://github.com/desktop/desktop/pull/12165
- Improvement to copy branch/repo names from context menu: https://github.com/desktop/desktop/pull/12178

I left https://github.com/desktop/desktop/pull/12327 out because it was done on top of a drag&drop refactor that came before 2.8.2, so probably not worth redoing the fix before 2.9.0.

## Release checklist

- [x] Check to see if there are any errors in Sentry that have only occurred since the last production release
- [x] Verify that all feature flags are flipped appropriately
- [x] If there are any new metrics, ensure that central and desktop.github.com have been updated

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.8.3": [
+      "[Fixed] Rename a repo and then create and clone a repo with that repo's previous name now points to a new repo not the renamed one - #3855",
```
Comment (tidy-dev) [inline changelog.json]: Ugg.. That is a mouth full.... if I do say so myself..

Comment (tidy-dev) [reply changelog.json]: Cloning a repository with the same name as a previously cloned repository that has been renamed no longer points to renamed repository.

Is that better?

Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] Renamed repositories no longer clash with new repositories of the same name - #3855",
```

**Review by billygriffin — DISMISSED** [review]
Just one minor suggestion and looks great!

**Review by tidy-dev — APPROVED** [review]
✨

**Pull request merged**